packages/create-vite/template-svelte/src/main.js

```diff
@@ -1,3 +1,4 @@
+import './style.css'
```

Comment:
`template-svelte/README.md`, `template-svelte-ts/README.md` might need to be updated.

Reply:
What should we update in the READMEs?

Reply:
The time I reviewed I thought some parts might need to be updated, but now I looked again and found nothing 😅
Please forget about it.